pennylane/transforms/qcut.py

```diff
@@ -18,16 +18,18 @@
 
 import copy
 import string
-from typing import Sequence, Tuple
+from itertools import product
+from typing import List, Sequence, Tuple
 
 from networkx import MultiDiGraph, weakly_connected_components
 
 import pennylane as qml
-from pennylane import apply
+from pennylane import Hadamard, Identity, PauliX, PauliY, PauliZ, S, apply, expval
+from pennylane.grouping import string_to_pauli_word
 from pennylane.measure import MeasurementProcess
-from pennylane.operation import Operation, Operator, Tensor
+from pennylane.operation import Expectation, Operation, Operator, Tensor
 from pennylane.ops.qubit.non_parametric_ops import WireCut
-from pennylane.tape import QuantumTape
+from pennylane.tape import QuantumTape, stop_recording
```

Comment:
We don't have to import everything from PL, we can do `qml.Hadamard()` etc. Maybe some key, regularly used functionality we can import directly.